cosmoz-treenode-navigator.html

```diff
@@ -66,14 +66,14 @@
 		</style>
 		<div id="header">
 			<h3 class="layout horizontal center wrap">
-				<a href="#" data-path="" on-click="openNode"><iron-icon icon="home"></iron-icon></a>
+				<a data-path="" on-click="openNode"><iron-icon icon="home"></iron-icon></a>
```

Comment:
Seems we lose the "hand" cursor after that change.
Also, though that's not priority number 1, we lose keyboard accessibility to this element.